### PR TITLE
Pvt/dell ctl volume list

### PR DESCRIPTION
**Pull request by mbasha-dell**
# Description
Documentation update for dellctl driver includes:

- Update for dellctl volume list feature

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/408|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/references/cli/_index.md [hunk 1]
```diff
@@ -782,3 +783,29 @@ Encryption rekey status with name of the rekey object
 # dellctl encryption rekey-status myrekey
  INFO Status of rekey request myrekey = completed
 ```
+### dellctl volume list
+
+Lists local volumes metadata for a given tenant.
```
Comment (sharmilarama) [inline content/docs/references/cli/_index.md:788]: Please indicate that this command is currently only supported for Powerflex volumes.

Comment (mbasha-dell) [reply content/docs/references/cli/_index.md]: Thanks @sharmilarama, this has been updated

**Comment by bharathsreekanth** [comment]
This should be merged to release-1.5.1 instead of main if i am not mistaken.

**Comment by mbasha-dell** [comment]
Closing this PR based on comment from @bharathsreekanth. Creating a new PR from pvt/dll-ctl-volume-list into release-1.5.1.

**Pull request closed without merging**